syntax = "proto3";

option go_package = ".;pb";

option java_multiple_files = true;
option java_package = "nxu.it";
option java_outer_classname = "HelloGrpcProto";

package hello;

service HelloGrpc {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}